### DISCUSSION TITLE
How to remove infinity scroll?

### DISCUSSION THREAD

**Discussion by minlapyae** [idiotWu/smooth-scrollbar#545] [Q&A]
When reach to the end, it's adding more spaces but If I remove canvas (particles star), it's working fine. that canvas is outside of the smooth scrollbar. I want to use a canvas as background.  How do I fix Infinite scrolling issue?
 
 
 [infinite.webm](https://github.com/idiotWu/smooth-scrollbar/assets/122955892/71a44f12-dc64-4572-b92f-295018afd4b9)

**Comment by sadeghbarati** [comment]
Can You share an online demo?
